static/js/containers/UserMenu.js

```diff
@@ -29,7 +29,12 @@ class LoginButton extends React.Component {
         <SplitButton
           title={title}
           bsStyle="danger"
-          id="logout-button">
+          id="user-menu">
+          <MenuItem
+            href={`/users/${profile.username}`}
+            eventKey="users_page">
```

Comment:
the `href` attribute on `MenuItem` will set a normal `<a>` html tag, which we actually don't want. Since we're using react-router on the frontend, we want to use something that will trigger a react-router route transition when clicked, instead of a full page refresh. `LinkContainer` is a little hack for doing just this with react-bootstrap UI components.